board/safety/safety_subaru.h

```diff
@@ -1,14 +1,21 @@
-#define SUBARU_STEERING_LIMITS_GENERATOR(steer_max, rate_up, rate_down)          \
-  {                                                                              \
-    .max_steer = (steer_max),                                                    \
-    .max_rt_delta = 940,                                                         \
-    .max_rt_interval = 250000,                                                   \
-    .max_rate_up = (rate_up),                                                    \
-    .max_rate_down = (rate_down),                                                \
-    .driver_torque_factor = 50,                                                  \
-    .driver_torque_allowance = 60,                                               \
-    .type = TorqueDriverLimited,                                                 \
-  }                                                                              \
+#define SUBARU_STEERING_LIMITS_GENERATOR(steer_max, rate_up, rate_down)               \
+  {                                                                                   \
+    .max_steer = (steer_max),                                                         \
+    .max_rt_delta = 940,                                                              \
+    .max_rt_interval = 250000,                                                        \
+    .max_rate_up = (rate_up),                                                         \
+    .max_rate_down = (rate_down),                                                     \
+    .driver_torque_factor = 50,                                                       \
+    .driver_torque_allowance = 60,                                                    \
+    .type = TorqueDriverLimited,                                                      \
+    /* the EPS will temporary fault if the steering rate is too high, so we cut the   \
+      the steering torque every 7 frames for 1 frame if the steering rate is high */  \
```

Comment:
```suggestion
    // the EPS will temporary fault if the steering rate is too high, so we cut the   \
    // the steering torque every 7 frames for 1 frame if the steering rate is high   \
```

Reply:
this is a quirk of the macros, everything is actually on one line when it's expanded, so if we use //, then everything after that will be commented out

Reply:
I haven't found a way to have proper multiline macros